app/Http/Controllers/BotController.php

```diff
@@ -5,14 +5,21 @@
 use Auth;
 use App\Models\Bot;
 use Illuminate\Http\Request;
+use App\Repositories\Interfaces\BotRepositoryInterface as BotRepository;
 
 class BotController extends Controller
 {
+	private $botRepository;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Http/Controllers/BotController.php</br>**Line**: 12</br>Spaces must be used to indent lines; tabs are not allowed